### PR TITLE
Implement User Deletion Workflow for Inactive Collaborators

### PR DESCRIPTION
**Pull request by sukeshreddyg**
## A reference to the issue / Description of it

This PR implements the workflow for scheduling user deletion for inactive collaborators whose inactivity is more than or equal to 180 days. #6390 

## How does this PR fix the problem?

This PR adds the necessary logic and steps to identify inactive collaborators and raise a GitHub issue for their deletion. By automating this process, we ensure that inactive collaborators are promptly identified and managed, enhancing the security and efficiency of AWS accounts.

### CODE REVIEW

**Review by mikereiddigital — APPROVED** [review]
lg2m

**Review by mikereiddigital — DISMISSED** [review]
lg2m

File: .github/workflows/check-inactive-collaborators.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   workflow_dispatch: 
 
 env:
+    GH_TOKEN: ${{ secrets.github_ci_user_pat }}
```
Comment (dms1981) [inline .github/workflows/check-inactive-collaborators.yml:7]: Do you specifically need to use the GitHub CI User PAT, or can you use automatic token authentication? (https://docs.github.com/en/actions/security-guides/automatic-token-authentication)

Comment (sukeshreddyg) [reply .github/workflows/check-inactive-collaborators.yml]: I tried both `github.token` and `secrets.GITHUB_TOKEN`, but neither of them worked.

**Pull request closed without merging**